pkg/namestore/namestore.go

```diff
@@ -115,3 +132,21 @@ func (x *nameStore) Rename(oldName, id, newName string) error {
 	}
 	return lockutil.WithDirLock(x.dir, fn)
 }
+
+func GetPrintableContainerName(containerLabels map[string]string) string {
```

Comment:
```suggestion
func PrintableContainerName(containerLabels map[string]string) string {
```

https://go.dev/doc/effective_go#Getters

Reply:
thanks